src/test/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKAnnotationPluginDescriptorUnitTest.java

```diff
@@ -0,0 +1,575 @@
+
+        Assert.assertFalse(vae.getInfoAnnotations().isEmpty());
+        Assert.assertTrue(vae.getInfoAnnotations().stream().noneMatch(a -> a.getClass().getSimpleName().equals(DepthPerSampleHC.class.getSimpleName())));
+        Assert.assertTrue(vae.getInfoAnnotations().stream().anyMatch(a -> a.getClass().getSimpleName().equals(BaseQualityRankSumTest.class.getSimpleName())));
+
+        // Asserting that the default set InbreedingCoeff has been overridden
+        Assert.assertEquals(Double.valueOf((String) ( vae.annotateContext(inbreedingCoefficientVC, new FeatureContext(),null, null, s -> true))
+                        .getAttribute(GATKVCFConstants.INBREEDING_COEFFICIENT_KEY)),
+                -0, 0.001, "InbreedingCoefficientScores");
+    }
+
+    @Test
+    public void testNoAnnotations(){
+        CommandLineParser clp = new CommandLineArgumentParser(
+                new Object(),
+                Collections.singletonList(new GATKAnnotationPluginDescriptor(null, null)),
+                Collections.emptySet());
+        String[] args = {};
+        clp.parseArguments(nullMessageStream, args);
+        List<Annotation> annots = instantiateAnnotations(clp);
+        final VariantAnnotatorEngine vae = new VariantAnnotatorEngine(annots, null, Collections.emptyList(), false);
+        Assert.assertTrue(vae.getGenotypeAnnotations().isEmpty());
+        Assert.assertTrue(vae.getInfoAnnotations().isEmpty());
+    }
+
+    @Test
+    public void testGetClassForPluginHelp(){
+        CommandLineParser clp = new CommandLineArgumentParser(
+                new Object(),
+                Collections.singletonList(new GATKAnnotationPluginDescriptor(null, null)),
+                Collections.emptySet());
+        String[] args = {};
+        clp.parseArguments(nullMessageStream, args);
+
+        final GATKAnnotationPluginDescriptor pluginDescriptor = clp.getPluginDescriptor(GATKAnnotationPluginDescriptor.class);
+
+        Assert.assertTrue(pluginDescriptor.getClassForPluginHelp(RMSMappingQuality.class.getSimpleName()).isInstance(new RMSMappingQuality()));
+        Assert.assertTrue(pluginDescriptor.getClassForPluginHelp("Foo")==null);
+    }
+
+    @Test
+    public void testIncludeDefaultExcludeIndividual(){
+        CommandLineParser clp = new CommandLineArgumentParser(
+                new Object(),
+                Collections.singletonList(new GATKAnnotationPluginDescriptor(Collections.singletonList(new Coverage()), null)),
+                Collections.emptySet());
+        String[] args = {"-AX", Coverage.class.getSimpleName()};
+        clp.parseArguments(nullMessageStream, args);
+        List<Annotation> annots = instantiateAnnotations(clp);
+        final VariantAnnotatorEngine vae = new VariantAnnotatorEngine(annots, null, Collections.emptyList(), false);
+        Assert.assertTrue(vae.getGenotypeAnnotations().isEmpty());
+        Assert.assertTrue(vae.getInfoAnnotations().isEmpty());
```

Comment:
Rather than using VAE,  I think its preferable to just test that the return from `instantiateAnnotations` is empty ? Same comment applies to the others below.

Reply:
The reason i did it this way was partially to assert that the VariantAnnotatorEngine was properly discriminating between genotype and InfoFieldAnnotations (which used to be done by its annotation manager)